packages/devextreme/js/__internal/ui/chat/avatar.ts

```diff
@@ -99,14 +99,25 @@ class Avatar extends Widget<Properties> {
     return result;
   }
 
-  _getInitials(name: string | undefined): string {
-    if (isDefined(name)) {
-      return String(name).charAt(0).toUpperCase();
+  _getInitials(name: string | number | undefined): string {
+    if (isString(name) || isNumeric(name)) {
+      const splitValue = String(name).trim().split(' ');
+
+      const firstInitial = this._getFirstChar(splitValue[0]);
+      const secondInitial = this._getFirstChar(splitValue[1]);
+
+      const result = `${firstInitial}${secondInitial}`;
```

Comment:
```suggestion
      return `${firstInitial}${secondInitial}`;
```

Reply:
I prefer to output the result before the function returns it, it's more convenient for debugging.